spatialmath/capsule_test.go

```diff
@@ -37,3 +37,52 @@ func TestBoxCapsuleCollision(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, dist, test.ShouldAlmostEqual, -29.69, 1e-3)
 }
+
+func TestCapsuleIntersectWithPlane(t *testing.T) {
+	c := makeTestCapsule(NewZeroOrientation(), r3.Vector{0, 0.1, 0.1}, 1, 16.75).(*capsule)
+	points, err := CapsuleIntersectionWithPlane(c, r3.Vector{0, 1, 0}, r3.Vector{1, 0, 0}, 32)
+	test.That(t, err, test.ShouldBeNil)
+
+	expectedPoints := []r3.Vector{
+		{1.000000000000000000000000, 0.100000000000000005551115, -8.275000000000000355271368},
```

Comment:
I think this test will be less flaky if we go to fewer bits of precision

Reply:
Good point, will update

Reply:
Actually, since I'm using ApproxEqual anyway, flakes should be a nonissue, but it would certainly become easier to read.